Anature/src/application/controllers/overworld_cells/PathOneController.java

```diff
@@ -0,0 +1,46 @@
+package application.controllers.overworld_cells;
+
+import application.LoggerStartUp;
+import application.enums.Direction;
+import application.enums.WarpPoints;
+import application.views.overworld_cells.PathOneCell;
+import javafx.scene.input.KeyEvent;
+
+public class PathOneController extends AbstractController
+{
+	public PathOneController(LoggerStartUp logger, PathOneCell view)
+	{
+		super(logger, view);
+	}
+
+	@Override
+	protected void timerHook()
+	{
+		
+	}
+
+	@Override
+	protected void keyPressHook(KeyEvent event)
+	{
+		
+	}
+
+	@Override
+	public void movePlayer(WarpPoints warpPoint)
+	{
+		if(warpPoint == null)
+			return;
+		
+		switch(warpPoint)
+		{
+			case Path_1_Starter_Town_Exit:
+				mPlayer.setX(45);
+				mPlayer.setY(292);
+				mView.setPlayerFacing(Direction.Right);
+				break;
+
+			default:
+				break;
+		}
+	}
+}
```

Comment:
Hey, why are these methods overrode but never implemented? Is it because those methods are unique for each scene?

Reply:
Ok, I saw why that was the case later in the PR. Looks good! 👍